code/backend/FakeMyResume/Program.cs

```diff
@@ -52,7 +52,7 @@
             {
                 policy.AllowAnyMethod();
                 policy.WithHeaders("Authorization", "Accept", "Referer", "User-Agent", "Content-Type");
-                policy.WithOrigins("https://localhost:7122");
+                policy.WithOrigins("https://localhost:7122", "https://localhost:4200");

```

Comment:
This will not be needed as going forward you should be testing while building angular and running it from visual studio

Reply:
done thanks